libraries/clchain/wasi-polyfill/eosio_assert.cpp

```diff
@@ -0,0 +1,16 @@
+#include <string.h>
+#include "polyfill_helpers.hpp"
+
+extern "C"
+{
+   [[clang::import_name("eosio_assert_message"), noreturn]] void
+   eosio_assert_message(uint32_t, const char* msg, uint32_t len)
+   {
+      polyfill::abort_message(msg, len);
```

Comment:
Not checking this condition?

Reply:
The wrappers only call it with a false condition, plus expect it to have `[[noreturn]]`